admin/python/lsst/qserv/admin/nodeAdmin.py

```diff
@@ -84,7 +84,7 @@ class _SSHCommand(object):
     # - known_hosts file has tendency to become inconsistent, ignore it
     sshOptions = ['StrictHostKeyChecking=no']
 
-    def __init__(self, host, cwd=None):
+    def __init__(self, host, cwd=None, kerberos=False, user=None):
```

Comment:
kerberos and user are documented in a later commit (See "File changed" tab above to get the whole updates)